### PR TITLE
Adds new accessory and beret options to the Tajara loadout

### PR DESCRIPTION
**Pull request by Alberyk**
This pr adds some accessories and berets to represent some tajaran organizations and their supporters.

### CODE REVIEW

**Review by SleepyGemmy — APPROVED** [review]
code looks fine. grammar looks fine.

**Review by Arrow768 — APPROVED** [review]

File: code/modules/clothing/head/xenos/tajara.dm [hunk 1]
```diff
@@ -53,31 +53,41 @@
...
 
+/obj/item/clothing/head/beret/tajaran/nka
+	name = "new kingdom naval beret"
+	desc = "A formal black beret with a blue band. This is worn by NKA naval servicemen and crewmen such as the Imperial Marines."
+	icon_state = "navalberetblue"
+	item_state = "navalberetblue"
+
+/obj/item/clothing/head/beret/tajaran/nka/officer
+	name = "new kingdom naval officer beret"
+	desc = "A formal black beret with a golden band. This is worn by members of the NKA naval officer corps. These are prised in the New Kingdom thanks to the Navy's popularity."
```
Comment (Greenjoe12345) [inline code/modules/clothing/head/xenos/tajara.dm:84]: prised > prized

Comment (NonQueueingMatt) [inline code/modules/clothing/head/xenos/tajara.dm:84]:
```suggestion
	desc = "A formal black beret with a golden band. This is worn by members of the NKA naval officer corps. These are prized in the New Kingdom thanks to the Navy's popularity."
```

File: code/modules/clothing/under/accessories/xeno/tajara.dm [hunk 2]
```diff
@@ -339,3 +339,104 @@
 	desc = "A hand carved charm of one of the mythical tajani."
 	desc_fluff = "Tajani, also known as 'short people' in basic, are good-willed tiny elder Tajara who serve as guardians of nature and homes. \
 	Hand carved charms of them is considered a symbol of luck and as such many superstitious tajara keeps one around."
+
+/obj/item/clothing/accessory/tajaran/tanker_pin
+	name = "golden sun pin"
+	desc = "Given to all members of the Hro'rammad tank corps is this sun pin. It is considered a symbol of how the tanker corps is the wrath of the God S'rendarr. It's rays are sharp not \
```
Comment (NonQueueingMatt) [inline code/modules/clothing/under/accessories/xeno/tajara.dm:345]:
```suggestion
	desc = "Given to all members of the Hro'rammad tank corps is this sun pin. It is considered a symbol of how the tanker corps is the wrath of the God S'rendarr. Its rays are sharp not \
```

File: code/modules/clothing/under/accessories/xeno/tajara.dm [hunk 3]
```diff
@@ -339,3 +339,104 @@
...
+	flippable = TRUE
+
+	drop_sound = 'sound/items/drop/ring.ogg'
+	pickup_sound = 'sound/items/pickup/ring.ogg'
+
+//dpra party badges
+
+/obj/item/clothing/accessory/tajaran/abparty_pin
+	name = "adhomaian blue party pin"
+	desc = "A pin of the Adhomaian Blue Party. This symbol is a white hoe on a blue background representing the party's dedication to environmentalism and ruralism."
```
Comment (NonQueueingMatt) [inline code/modules/clothing/under/accessories/xeno/tajara.dm:376]:
```suggestion
	desc = "A pin of the Adhomian Blue Party. This symbol is a white hoe on a blue background representing the party's dedication to environmentalism and ruralism."
```

File: code/modules/clothing/under/accessories/xeno/tajara.dm [hunk 4]
```diff
@@ -339,3 +339,104 @@
...
+	Nated'Hakhan, stripping away any semblance of democracy from the Democratic People's Republic. Members of this party oppose any form of peace or negotiation with the other Tajaran factions, \
+	or diplomacy with alien powers. They are known for hosting the most radical elements of DPRA's political scenario."
+	flippable = TRUE
+
+	drop_sound = 'sound/items/drop/ring.ogg'
+	pickup_sound = 'sound/items/pickup/ring.ogg'
+
+/obj/item/clothing/accessory/tajaran/ftpparty_pin
+	name = "free tajaran people party pin"
+	desc = "A pin of the Free Tajaran's People Party. The two white rifles represent the party's fervor in crushing the two ideological enemies of the Tajara: Royalism and Hadiism. "
```
Comment (NonQueueingMatt) [inline code/modules/clothing/under/accessories/xeno/tajara.dm:404]:
```suggestion
	desc = "A pin of the Free Tajaran's People Party. The two white rifles represent the party's fervor in crushing the two ideological enemies of the Tajara: Royalism and Hadiism."
```

**Review by NonQueueingMatt — APPROVED** [review]

File: code/modules/client/preference_setup/loadout/loadout_xeno/tajara.dm [hunk 5]
```diff
@@ -358,4 +363,36 @@
...
+	whitelisted = list(SPECIES_TAJARA, SPECIES_TAJARA_ZHAN, SPECIES_TAJARA_MSAI)
+	flags = GEAR_HAS_DESC_SELECTION
+
+/datum/gear/accessory/dpra_party_pin/New()
+	..()
+	var/list/card = list()
+	card["national adhomai workers party pin"] = /obj/item/clothing/accessory/tajaran/nawparty_pin
+	card["free tajaran people party pin"] = /obj/item/clothing/accessory/tajaran/ftpparty_pin
+	card["followers of Nated party pin"] = /obj/item/clothing/accessory/tajaran/fonparty_pin
+	card["adhomaian blue party pin"] = /obj/item/clothing/accessory/tajaran/abparty_pin
```
Comment (SleepyGemmy) [inline code/modules/client/preference_setup/loadout/loadout_xeno/tajara.dm:381]:
```suggestion
	card["adhomian blue party pin"] = /obj/item/clothing/accessory/tajaran/abparty_pin
```

**Review by SleepyGemmy — CHANGES_REQUESTED** [review]

**Pull request merged**